lib/roots_web/schema/recipe_types.ex

```diff
@@ -0,0 +1,32 @@
+defmodule RootsWeb.Schema.RecipeTypes do
+  use Absinthe.Schema.Notation
+
+  import Absinthe.Resolution.Helpers, only: [dataloader: 1]
+
+  alias RootsWeb.{Data, Resolvers}
+
+  @desc "Recipe"
+  object :recipe do
+    field :id, :id
+    field :description, :string
+    field :instructions, :string
+    field :title, :string
+    field :author, :string
+
+    field :cookbook, :cookbook, resolve: dataloader(Data)
```

Comment:
Recipe types file looks good!
One thing I think we can do now is uncomment line 15 in cookbook_types to tie the relationships between recipe and cookbook together now. Same line as you have on 17 for ingredients

Reply:
Got it - I'll add the relationship between recipe and cookbook, and re-request your review.